<?xml version="1.0"?>
<launch>
  <arg name="checkpoint_path" default="graspnet_checkpoints/checkpoint.tar" />
  <arg name="num_grasp_candidates" default="5" />
  <arg name="pointcloud_topic" default="/camera/depth/points" />
  <arg name="rgb_topic" default="/camera/color/image_raw" />
  <arg name="camera_info_topic" default="/camera/color/camera_info" />
  <arg name="sync_slop" default="0.08" />
  <arg name="use_detection_roi" default="true" />
  
  <node name="grasp_estimator" pkg="perception_grasp" type="grasp_estimator_node.py" output="screen">
    <param name="checkpoint_path" value="$(arg checkpoint_path)" />
    <param name="num_grasp_candidates" value="$(arg num_grasp_candidates)" />
    <param name="pointcloud_topic" value="$(arg pointcloud_topic)" />
    <param name="rgb_topic" value="$(arg rgb_topic)" />
    <param name="camera_info_topic" value="$(arg camera_info_topic)" />
    <param name="sync_slop" value="$(arg sync_slop)" />
    <param name="use_detection_roi" value="$(arg use_detection_roi)" />
  </node>
</launch>
